tests/ui/iterators/ZST-nthback.rs

```diff
@@ -0,0 +1,10 @@
+//@ run-pass
```

Comment:
This still will not be a test we run under miri, I believe.

We do run the standard library's tests under miri, however.